tests/dummy/app/pods/demo/frost-fixed-table/template.hbs

```diff
@@ -62,8 +73,8 @@
         )
         (hash
           className='real-name-col'
-          label='Real Name'
-          propertyName='realName'
+          label='Team Affiliations'
+          propertyName='teamAffiliations'
```

Comment:
There were duplicate ones to show the scrolling. That said, I just noticed that I also gave it the class name of `real-name-col`, which obviously isn't right.